nss-cohort-37/Workforce-Green-Monkeys
# Et employees

Testing Needed

Ticket #1 Employee View needs review.  

1. Please Click employees link in the navigation bar and review the given employees listing

Verify that the list of shows First Name, Last Name, and Department

 [ ] New feature (non-breaking change which adds functionality)
 [ ] I have performed a self-review of my own code
[ ] I have added test instructions that prove my fix is effective or that my feature works


Ticket #2 Add Employee

1. Please Click employees link in the navigation bar 
2. Please Click Create New employee
3. Enter First Name, Last Name, email, check if they are a Supervisor or leave blank
4. Select the department they work in from the drop list
5. Select the computer they will be assigned from the drop list
6. Press Create to add to database
7. Return to employee listing and verify the new employee is added

 [ ] New feature (non-breaking change which adds functionality)
 [ ] I have performed a self-review of my own code
[ ] I have added test instructions that prove my fix is effective or that my feature works

Ticket #4 Edit Employee

1. Please Click employees link in the navigation bar 
2. Please Click Edit on a specific employee 
3. Change the first and/or last name
4. Change the department
5. Change the assigned computer
6. Press SAVE button
7. Return to employee listing and verify the new employee is updated.
8. Go to computer table in BangazonWorkforce database to verify the employee is assigned the new computer.

[ ] New feature (non-breaking change which adds functionality)
 [ ] I have performed a self-review of my own code
[ ] I have added test instructions that prove my fix is effective or that my feature works
